torch/testing/_internal/common_utils.py

```diff
@@ -5567,3 +5570,17 @@ def load_inline(*args, **kwargs):
         return func(*args, load_inline=load_inline, **kwargs)
 
     return wrapper
+
+
+HAS_CUDA = torch.cuda.is_available()
```

Comment:
In common_device_type.py, there are TEST_CUDA, TEST_XPU...,   could we unify this variable?